ui/apps/platform/cypress/integration/dashboard.test.js

```diff
@@ -79,52 +115,43 @@ describe('Dashboard page', () => {
 
     it('should display violations by cluster chart for single cluster', () => {
         cy.intercept('GET', api.alerts.countsByCluster, {
-            fixture: 'alerts/countsByCluster-single.json',
-        });
-
+            body: alertsSummaryCountsByCluster1,
+        }).as('getAlertsCountsByCluster');
         visitMainDashboard();
+        cy.wait('@getAlertsCountsByCluster');
 
-        cy.get(selectors.sectionHeaders.violationsByClusters).next().as('chart');
-
-        cy.get('@chart').within(() => {
-            cy.get(selectors.chart.xAxis).should('contain', 'Kubernetes Cluster 0');
-            cy.get(selectors.chart.grid).spread((grid) => {
-                // from alerts fixture : low = 2, medium = 1, therefore medium's height should be twice less
-                const { height } = grid.getBBox();
-                cy.get(selectors.chart.lowSeverityBar).should('have.attr', 'height', `${height}`);
-                cy.get(selectors.chart.medSeverityBar).should(
-                    'have.attr',
-                    'height',
-                    `${height / 2}`
-                );
-            });
+        cy.get(selectors.chart.xAxis).should('contain', 'Kubernetes Cluster 0');
+
+        // For future design of main dashboard: Accessible data does not need tricky assertions.
```

Comment:
Thanks for this comment as a reminder. Lets hope PatternFly/Victory charts are more accessible.